dpti/equi.py

```diff
@@ -600,6 +601,35 @@ def post_task(iter_name, natoms=None, is_water=None):
     return info_dict
 
 
+def run_task(task_name, machine_file):
+    task_dir_list = [task_name]
+    work_base_dir = os.path.join(os.getcwd())
```

Comment:
Consider using pathlib for handling file paths, it's more reliable and readable than using os.path. [medium]